packages/styles/src/components/button.scss

```diff
@@ -42,7 +42,7 @@
 
 // Primary
 .btn-primary {
-  border-color: transparent;
+  border-color: rgba(var(--post-contrast-color-rgb), 0.8);
```

Comment:
L34 and L45 are to address "Change all colors from transparent colors to full-tone colors (background, color, border)" task. On this one, I'm not sure if this is the goal of this task.

Reply:
Ahhh ok thanks! I didn't know that --post-gray-# is actually dynamic according to the theme.